src/prefect/cli/block.py

```diff
@@ -109,3 +112,56 @@ async def register(
         "\n To configure the newly registered blocks, "
         "go to the Blocks page in the Prefect UI.\n"
     )
+
+
+@blocks_app.command()
+async def ls():
+    """
+    View all configured Blocks.
+    """
+    async with get_client() as client:
+        blocks = await client.read_block_documents()
+
+    table = Table(
+        title="Blocks", caption="List Block Types using `prefect block type ls`"
+    )
+    table.add_column("ID", style="cyan", no_wrap=True)
+    table.add_column("Name", style="blue", no_wrap=True)
+    table.add_column("Type", style="blue", no_wrap=True)
```

Comment:
If possible, I think it'd be good to also display block document slugs which are {block_type_slug}/{block_doucment_name}
```suggestion
    table.add_column("ID", style="cyan", no_wrap=True)
    table.add_column("Name", style="blue", no_wrap=True)
    table.add_column("Type", style="blue", no_wrap=True)
    table.add_column("Slug", style="blue, no_wrap=True)
```